src/store/construct.ts

```diff
@@ -41,10 +41,32 @@ export interface StoreState {
   messages: List<Message>;
 }
 
+const settingsMigrations = {
+  1: (state: any) => {
+    const oldViewSettings = { ...state.viewSettings };
+    const newSettings = {
+      defaultViewMode: "last",
+      lastViewMode: oldViewSettings.viewMode ?? "true",
+      editorFontFamily: "monospace",
+      viewerFontFamily: "regular",
+    };
+    if (oldViewSettings.viewMode) { delete oldViewSettings.viewMode }
+    const viewSettings: SettingsType["viewSettings"] = {
+      ...newSettings,
+      ...oldViewSettings,
```

Comment:
I think it's better to be explicit about setting only the keys you expect to have rather than have the `delete oldViewSettings.viewMode` and the spreading operators here. It shouldn't be a lot of fields anyway.